OutOfSchool/OutOfSchool.WebApi/Controllers/V1/WorkshopMementoController.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using OutOfSchool.BusinessLogic.Models.Workshops.IncompletedWorkshops;
+using OutOfSchool.BusinessLogic.Services.Memento.Interfaces;
+
+namespace OutOfSchool.WebApi.Controllers.V1;
+
+/// <summary>Controller with CRUD operations for WorkshopWithRequiredPropertiesDto entity.</summary>
+[ApiController]
+[ApiVersion("1.0")]
+[Route("api/v{version:apiVersion}/[controller]/[action]")]
+public class WorkshopMementoController(IMementoService<WorkshopWithRequiredPropertiesDto> mementoService)
```

Comment:
Are you sure it should contain a `memento` as a part of the name ?

Reply:
I think controller name should contain **memento** in it, because in API may contain another controller with **memento**, for example - ProviderMementoController